src/eradiate/_mode.py

```diff
@@ -68,33 +67,45 @@ class ModeFlags(enum.Flag):
 
 # Map associating a mode ID string to the corresponding class
 # (aliased in public API section)
-_mode_registry = {
```

Comment:
From what I understand, the end user API does not change thanks to this mapping am I right?

Reply:
Indeed